README.md

```diff
@@ -112,15 +112,13 @@ RQ: Do the certainty scores in CLIP3 predictions correspond to the actual catego
 ## Methods, results
 To answer the above, I examined the 'first-choice category' values in the primary data, focussing mainly on the highest and the lowest confidence values to scan the data for possible patterns. To interpret any tendencies found, I used preliminary evaluation data produced during the Hackathon ('Secondary data' above), and close-reading of the images themselves. 
```

Comment:
I just learned that *focussing* is correct and I hate it with a passion 🔥  I could get over the antiquated feel, but not over the hint of *cussing*